### PR TITLE
Feature/trigger intent by sending followup event from webhook

### PR DESCRIPTION
**Pull request by cesaruve**
Continuing ndjido job in PR #84

### CODE REVIEW

**Comment by cesaruve** [comment]
@ndjido  are you ok with this PR?

File: test/dialogflow-app-test.js [hunk 1]
```diff
@@ -3670,4 +3670,42 @@ describe('DialogflowApp', function () {
...
   });
+
+  /**
+   * Describes the behavior for DialogflowApp askForUpdatePermission method in v1.
+   */
+  describe('#followUpEvents', function () {
+    let mockRequest, app;
+
+    beforeEach(function () {
+      mockRequest = new MockRequest(headerV1, dialogflowAppRequestBodyLiveSession);
```
Comment (smishra2) [inline test/dialogflow-app-test.js]: Change `headerV1` to `headerV2`, this might allow you to use camelCase in your tests, rather than snake_case

**Comment by smishra2** [comment]
Thanks for picking this up @cesaruve !

File: test/dialogflow-app-test.js (hunk 1 above)
Comment (cesaruve) [reply test/dialogflow-app-test.js]: ahhhh that's it! I was going mad.. thanks for your quick reply. Sorry for hacking the test, I didn't feel well when I did the commit.

Thank you for making such an amazing product.

**Comment by smishra2** [comment]
Will wait on approval from @ndjido before merging.

**Comment by smishra2** [comment]
Thanks @ndjido and @cesaruve, glad we were able to land this!

**Comment by c4ndybar** [comment]
This does not work for Dialogflow API v1.

**Comment by ndjido** [comment]
@c4ndybar I have it working. Could you please open a new issue and give a code example to we can help you?

**Pull request merged**